parmed/openmm/parameters.py

```diff
@@ -200,13 +204,22 @@ def write(self, dest, provenance=None, write_unused=True, separate_ljforce=False
             except KeyError:
                 warnings.warn('Some residue templates are using unavailable '
                               'AtomTypes', ParameterWarning)
+
+        # DEBUG
+        print('Determining valid patch combinations...')
```

Comment:
Change this to use `logging` instead of `print`

Reply:
Are there examples of logging output elsewhere I could model this on?